src/options.jl

```diff
@@ -0,0 +1,121 @@
+"""
+Options passed to `pgfplots` for various structures (`table`, `plot`, etc).
+
+Contents emitted in `key = value` form, or `key` when `value ≡ nothing`. Also
+see the [`@pgf`](@ref) convenience macro.
+"""
+const Options = OrderedDict
```

Comment:
Shouldn't this be `OrderedDict{Any, Any}` for it to be concrete?

Reply:
I don't understand what the advantage of it being concrete would be, can you please clarify?

This way I can use it for both the constructor (empty `Options()`) and dispatch. Perhaps I should use an `UnionAll` type explicitly?

Reply:
It probably doesn't matter. Would save dynamic dispatch when accessing the options field but it shouldn't really have any relevance.

Reply:
But I don't understand why you couldn't use it for a constructor and dispatch even if it was concrete?

Reply:
Since in the original code some methods were dispatched on `::OrderedDict`, I thought that the implicit expectation was that we would take any `<: OrderedDict`, hence the `UnionAll` type.

If that's not required, I can make it concrete, no problem.

Reply:
I think they will always be `{Any, Any}`.

Reply:
At least if we use the `Option` type everywhere, they will be by construction.